dataverse-webapp/src/main/webapp/editdatafiles.xhtml

```diff
@@ -60,10 +57,16 @@
 
                     <ui:include src="editFilesFragment.xhtml">
                         <ui:param name="editDatafilesPage" value="true"/>
+                        <ui:param name="isDatasetSection" value="false"/>
                         <ui:param name="createDataset" value="false"/>
                         <ui:param name="showFileButtonUpdate" value="true"/>
                         <ui:param name="version" value="#{EditDatafilesPage.workingVersion}"/>
                         <ui:param name="lockedFromEdits" value="#{EditDatafilesPage.lockedFromEdits}"/>
+                        <ui:param name="showFileUpload" value="#{EditDatafilesPage.showFileUploadFragment()}"/>
+                        <ui:param name="isMultiplyFilesUploadAllowed" value="#{EditDatafilesPage.allowMultipleFileUpload()}" />
+                        <ui:param name="fileUploadDisabled" value="#{!EditDatafilesPage.showFileUploadComponent()}" />
+                        <ui:param name="isUploadPermited" value="#{EditDatafilesPage.mode eq 'UPLOAD'}" />
+                        <ui:param name="redirectToEditdataFiles" value="" />
```

Comment:
If redirect is not used in this view then I 'think' you can remove param.

Reply:
It is used with specific value of ""